src/components/tasks/card/index.tsx

```
@@ -629,6 +616,36 @@
                 {showAssignButton() && <AssigneeButton />}
             </div>
 
+            <div className={styles.taskStatusEditMode}>
+                {isDevMode && (isEditable || isSelfTask) ? (
+                    <div className={styles['task-status-control']}>
+                        <button
+                            className={styles['extension-button']}
+                            onClick={() => setShowExtensionModal(true)}
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)